hvplot/converter.py

```diff
@@ -552,6 +553,7 @@ def __init__(
         invert=False,
         stacked=False,
         colorbar=None,
+        cticks=None,
```

Comment:
I think `cticks` should be added to the docstring and relevant page in the docs. Adding an example *somewhere* would also be nice.